config/default.yml

```diff
@@ -104,6 +104,11 @@ products:
     price: 5.09
   - name: "Mozilla Horizontal T-Shirt"
     price: 5.94
+  - name: 'OWASP SSL Advanced Forensic Tool (O-Saft)'
+    description: 'O-Saft is an easy to use tool to show information about SSL certificate and tests the SSL connection according given list of ciphers and various SSL configurations.'
+    price: 0.01
+    image: 'owasp_osaft.jpg'
+    useForProductTamperingChallenge: true
```

Comment:
Alternatively we could update all the tests to use ZAP or another tampering challenge product